apps/server/prisma/schema.prisma

```diff
@@ -192,3 +174,56 @@ model Subscription {
   isActive  Boolean
   createdAt DateTime @default(now())
 }
+
+model password_reset_tokens {
+  id          Int       @id @default(autoincrement())
+  user_id     Int?
+  reset_token String
+  expires_at  DateTime  @db.Timestamp(6)
+  used        Boolean?  @default(false)
+  created_at  DateTime? @default(now()) @db.Timestamp(6)
+  users       users?    @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction)
+}
+
+model social_accounts {
+  id               Int       @id @default(autoincrement())
+  user_id          Int?
+  provider         String    @db.VarChar(100)
+  provider_user_id String    @db.VarChar(255)
+  access_token     String?
+  refresh_token    String?
+  token_expires_at DateTime? @db.Timestamp(6)
+  created_at       DateTime? @default(now()) @db.Timestamp(6)
+  updated_at       DateTime? @default(now()) @db.Timestamp(6)
+  users            users?    @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@unique([provider, provider_user_id])
+}
+
+model user_sessions {
+  id            Int       @id @default(autoincrement())
+  user_id       Int?
+  session_token String
+  created_at    DateTime? @default(now()) @db.Timestamp(6)
+  expires_at    DateTime? @db.Timestamp(6)
+  last_active   DateTime? @default(now()) @db.Timestamp(6)
+  users         users?    @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction)
+}
+
+model users {
+  id                    Int                     @id @default(autoincrement())
+  email                 String                  @unique @db.VarChar(255)
+  password_hash         String?
+  first_name            String?                 @db.VarChar(100)
+  last_name             String?                 @db.VarChar(100)
+  created_at            DateTime?               @default(now()) @db.Timestamp(6)
+  updated_at            DateTime?               @default(now()) @db.Timestamp(6)
+  password_reset_tokens password_reset_tokens[]
+  social_accounts       social_accounts[]
+  user_sessions         user_sessions[]
+}
+
+enum ContributionTypes {
```

Comment:
Enums: Changed the enum values to uppercase (FINANCIAL, VOLUNTEER), following common enum naming conventions.